feat: add custom Dockerfile for Ubuntu 20 and Ubuntu 18 improvements (#195)
openvinotoolkit/docker_ci: dockerfiles/README.md

----------------

* Custom image with CPU, iGPU, VPU support  

> We prepared Dockerfiles for custom development OpenVINO™ from source. To learn more, follow:

Reviewer: ```suggestion
Use Dockerfiles to develop a custom version of OpenVINO™ from source. To learn more, follow:
```

Author: May be replace it to something like this:

You can use Dockerfiles from `build_custom` folders to build a custom version of OpenVINO™ from source codes for development
